tailwind.config.js

```diff
@@ -0,0 +1,78 @@
+/** @type {import('tailwindcss').Config} */
```

Comment:
I'd expect you to end up with merge conflicts in this file, since it's being added in multiple PRs, and it looks slightly different in each one.

As you merge each PR, it'll be a good idea to check this file and test your changes locally to make sure that you aren't losing any styles in the merge.

Reply:
Sounds good! Thank you :)